### DISCUSSION TITLE
Project status?

### DISCUSSION THREAD

**Discussion by gsakkis** [art049/odmantic#352] [General]
Hey @art049, wondering what's the status of this project? As of now there are 26 open pull requests (granted, about half are automated `dependabot`PRs) and commits have all but stopped since last September. Any plans on reviving it? If it's a matter of time/effort, I'd be happy to assist as co-maintainer on a voluntary basis.

**Comment by SYNchroACK** [comment]
@art049 would be nice to have some feedback here.

**Comment by art049** [comment]
Hey @gsakkis, awesome! Sorry, I haven't been very responsive lately.

If you're up for co-maintaining, that would be truly amazing.

I started working a little on pydantic v2 support. It will imply some quite big changes from the initial look I had to it. 
It will probably be a breaking change from pydantic v1 so it will be time for ODMantic v1 😀
I think the fixes can be included before this but it would probably be more interesting to work on big features just after.

Let me know what you think!

**Reply by Esposus** [reply]
@art049 Hi! Do you still support the library? Could you please add support for modern type annotations in Python 3.10+? And it seems like Pydanticv2 Field does not support  regex anymore and raises pydantic.errors.PydanticUserError: `regex` is removed. use `pattern` instead

**Comment by arnabJ** [comment]
Hey @art049, wondering what's the status of this project?

I really like this ODM and I am already using it on one live project and even created a Custom field type for it and released on PyPi.

I want to use it in another project but I am not sure if I should or look into some other ODM.

**Comment by parth7676** [comment]
Is the project still alive?
